plugins/convert_README_to_header.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Does this only work with bash then? I'd be slightly worried about support challenges on other OSes if we have to switch away.

Reply:
Works fine in sh too, while I was trying to fix it I changed it, will revert.